modules/core/src/v2/coins/eos.ts

```diff
@@ -124,6 +149,12 @@ export class Eos extends BaseCoin {
     return true;
   }
 
+  /**
+   * Get URL of a public node
+   */
+  getPublicNodeUrl(): string {
+    return 'https://bp.cryptolions.io';
```

Comment:
maybe a few different options for each network in case one is down? If that's too annoying we can do it later if it becomes a problem